Tests/DatadogBenchmarkTests/DataStorage/DataMigratorBenchmarkTests.swift

```diff
@@ -0,0 +1,57 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import XCTest
+@testable import Datadog
+
+@available(iOS 13.0, *)
+class DataMigratorBenchmarkTests: XCTestCase {
+    private let benchmarkOptions: XCTMeasureOptions = {
+        let options = XCTMeasureOptions()
+        options.invocationOptions = [.manuallyStart, .manuallyStop]
+        options.iterationCount = 5
+        return options
+    }()
+    private let benchmarkMetrics: [XCTMetric] = [XCTClockMetric(), XCTMemoryMetric()]
+
+    func testMigrating500FilesWithDeleteAllDataMigrator() throws {
+        let directory = try Directory(withSubdirectoryPath: UUID().uuidString)
+        defer { directory.delete() }
+
+        measure(metrics: benchmarkMetrics, options: benchmarkOptions) {
+            createFiles(in: directory, count: 500)
+
+            self.startMeasuring()
+            let migrator = DeleteAllDataMigrator(directory: directory)
```

Comment:
you do not want to create this before measuring ?

Reply:
I've been considering this carefully - creation of migrator is part of the actual migration work, so I consider it rather correct to be included in the benchmarked scope.